doc/Building.md

```diff
@@ -34,6 +34,24 @@ Install the prerequisites using DNF:
 sudo dnf install libstdc++-static SDL SDL-devel patch wget unzip git cmake luarocks autoconf nasm ragel gcc
 ```
 
+### Arch
+
+Install the prerequisites using PACMAN:
+
+```sh
+# The following may not be a complete list of required packages
+sudo pacman -S luarocks git ncurses
+```
+
+Also some manual changes have to be done:
+
+Arch only provides ncurses 6 now, but koreader needs `libncurses.so.5` and `libtinfo.so.5` but `ncurses` by default does not link them, so it has to be done manually:
+
+```sh
+sudo ln -sf /lib/libncursesw.so.6 /lib/libncurses.so.5
+sudo ln -sf /lib/libncursesw.so.6 /lib/libtinfo.so.5
+```
+
```

Comment:
I have an up to date arch install.
ncurses included (bash along with half of my packages rely on it in some way)
I have never needed to symlink anything so this section should not be required. (Maybe the android SDK needs it but that is not a koreader issue)

Reply:
cmake luarocks & git all depend on curl -> ca-certificates -> bash -> ncurses, and bash is part of the Arch base install. so it seems to me this section is superfluous.

The generic linux prerequisites should be enough.

Reply:
Why would Bash depend on ncurses? O.o

Bash itself may be an unwritten dependency though.